runnc-cont/runnc_cont.go

```diff
@@ -135,6 +137,14 @@ func run(nablarun string, unikernel string, tapName string,
 			fmt.Fprintf(os.Stderr, "Could not create %s: %v\n", tapName, err)
 			return 1
 		}
+	} else if inK8s {
+		// The tap device will get the IP assigned to the Docker
+		// container veth pair.
+		ip, gw, mask, err = network.CreateTapInterfaceDocker(tapName, "eth0")
```

Comment:
Add comment (a XXX) mentioning the error you get when using macvtaps. It's kind of mysterious to me why this is working and the macvtap not when using k8.